crates/aptos-ledger/src/lib.rs

```diff
@@ -143,17 +143,86 @@ pub fn get_app_name() -> Result<String, AptosLedgerError> {
     }
 }
 
-/// Returns the public key of your Aptos account in Ledger device
+/// Returns the next available account in Ledger starting from start_index to end_index, inclusive
+/// Note: We only allow a range of 20 for performance purpose
+///
+/// # Arguments
+///
+/// * `start_index` - The index to start fetch for accounts
+/// * `end_index` - The last index to fetch the accounts
```

Comment:
Could you leave a comment about whether this is an inclusive or exclusive range?